internal/build/phases.go

```diff
@@ -211,14 +211,19 @@ func prependArg(arg string, args []string) []string {
 }
 
 func (l *Lifecycle) Build(ctx context.Context, networkMode string, volumes []string, phaseFactory PhaseFactory) error {
+	args := []string{"-layers", layersDir,
+		"-app", appDir,
+		"-platform", platformDir,
+	}
+
+	if l.platformAPIVersion > "0.2" { // lifecycle did not support log level for build until platform api 0.3
```

Comment:
This should be using the semver library for comparison.

For example:

https://github.com/buildpacks/pack/blob/ff881aafd4883ceb7318324647b74fbbe014de22/internal/build/phases.go#L349-L352